src/api.ts

```diff
@@ -514,7 +514,7 @@ export type GasRecommendationResponse =
       message: string
     }
   | {
-      available: true
+      available: boolean
```

Comment:
I'm not sure if that makes sense. I would say if `available` is `false` we still want to return a `message` why it's not available, together with the remaining information?

Reply:
Let's make it optional then 😁